funding/manager.go

```diff
@@ -1715,6 +1737,9 @@ func (f *Manager) handleFundingOpen(peer lnpeer.Peer,
 			},
 		},
 		UpfrontShutdown: msg.UpfrontShutdownScript,
+		LocalNonce: &musig2.Nonces{
+			PubNonce: *msg.LocalNonce,
```

Comment:
The peer may not send `LocalNonce` which means it will be `nil` and then this will panic

Reply:
haha I just came here to comment this cause I just ran into this panic 🙃 nice catch!

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x1014a0c0c]

goroutine 311 [running]:
github.com/lightningnetwork/lnd/funding.(*Manager).handleFundingOpen(0x140001f2360, {0x101d42e68, 0x140008b9c00}, 0x1400014a300)
	/Users/elle/LL/lnd/funding/manager.go:1765 +0x1d1c
github.com/lightningnetwork/lnd/funding.(*Manager).reservationCoordinator(0x140001f2360)
	/Users/elle/LL/lnd/funding/manager.go:894 +0x334
created by github.com/lightningnetwork/lnd/funding.(*Manager).start
	/Users/elle/LL/lnd/funding/manager.go:719 +0x53c
```